data/tlg0086/tlg013/tlg0086.tlg013.1st1K-grc1.xml

```diff
@@ -0,0 +1,2035 @@
+                </respStmt>
+                <respStmt>
+                    <persName>Annette Gessner</persName>
+                    <resp>Project Assistant (University of Leipzig) 2015 - present</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Thibault Clérice</persName>
+                    <resp>Lead Developer (University of Leipzig)</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Bruce Robertson</persName>
+                    <resp>Technical Advisor</resp>
+                </respStmt>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Harvard College Library</authority>
+                <idno type="filename">tlg0086.tlg013.1st1K-grc1.xml</idno>
+                <availability>
+                    <p>Available under a Creative Commons Attribution-ShareAlike 4.0 International
+                        License</p>
+                </availability>
+                <date>2017</date>
+                <publisher>Harvard College Library</publisher>
+                <pubPlace>United States</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <listBibl>
+                    <biblStruct>
+                        <monogr>
+                            <title>Aristotle. On Sophistical Refutations. On Coming-to-be and
+                                Passing-away. On the Cosmos</title>
+                            <editor>
+                                <persName>
+                                    <name>E. S. Forster</name>
+                                </persName>
+                                <persName>
+                                    <name>D. J. Furley</name>
+                                </persName>
+                            </editor>
+                            <author ref="urn:cts:greekLit:tlg0086">Aristotle</author>
+                            <imprint>
+                                <publisher>William Heinemann</publisher>
+                                <pubPlace>London</pubPlace>
+                                <date>1955</date>
+                            </imprint>
+                            <biblScope unit="volume"/>
+                        </monogr>
+                        <ref
+                            target="https://archive.org/stream/onsophisticalref00arisuoft#page/162/mode/2up/search/On+sophistical+refutations"
+                            >Internet Archive</ref>
+                    </biblStruct>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>Text encoded in accordance with the latest EpiDoc standards</p>
+            <refsDecl n="CTS">
+                <cRefPattern matchPattern="(.+).(.+).(.+)" n="paragraph"
+                    replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1']/tei:div[@n='$2']/tei:p[@n='$3'])"/>
```

Comment:
As in the previous PR, I would suggest enclosing the `<p>` elements in `<div>` elements instead of using `<p>` directly as a citation unit.

Reply:
I actually think I woul just delete the paragraph level since the other version of the work only goes down to the chapter level. That would mean that their citation schemes will match up.